gulp/tasks/test.js

```diff
@@ -111,6 +122,17 @@ gulp.task( 'test:coveralls', function() {
     .pipe( $.coveralls() );
 } );
 
+// This task will only run on Travis
+gulp.task( 'test:a11y', function() {
+  spawn(
+    fsHelper.getBinary( 'wcag', '.bin' ),
+    _getWCAGParams(),
+    { stdio: 'inherit' } )
```

Comment:
Can we break the closing parens to a new line matching indent of `spawn` so that it's clear it's closing that function. This block is kind of hard to follow.